app/retail/templates/emails/new_bounty_expire_warning.html

```diff
@@ -20,32 +20,37 @@
 
 {% block content %}
 
-<h1>😕 {% trans "Funded Issue Expiring ..." %} 😕</h1>
+<h1>{% trans "Your Funded Bounty is Expiring" %}</h1>
 
 
-<p>
+<p style="color: #4A4A4A;">
     {% if is_owner %}
         {% blocktrans %}Heads up! The funded issue that you posted (below) is expiring in {{num}} {{unit}}.{% endblocktrans %}
     {% else %}
         {% blocktrans %}Heads up! The funded issue that you fulfilled (below) is expiring in {{num}} {{unit}}.{% endblocktrans %}
     {% endif %}
 
+    <br>
+    <br>
+    <img src="{% static "v2/images/expiring_clock.png"%}" style="padding: 20px 0;">
```

Comment:
would you hate me if we used svg here instead ? :P

Reply:
No I won't. But won't SVG finally get converted to PNG after [this](https://github.com/gitcoinco/web/pull/2106)? @mbeacom should we still be using SVG to maintain proper quality of image?